chanzuckerberg/idseq-web
# Add tooltips to metadata fields on upload or update pages

# Description

As title. See for example.

![image](https://user-images.githubusercontent.com/28797/74791828-6b67c900-5270-11ea-92d5-1e729b1b88bd.png)

![image](https://user-images.githubusercontent.com/28797/74791125-74f03180-526e-11ea-9515-a84ef821d42d.png)

# Notes

* Note these descriptions were shortened from a subset of the current stored descriptions of `MetadataField` by @oliviabholmes .
* I added tooltip for host organism
* Learn more goes to metadata dictionary page

# Tests

* Go to upload screen
* Hover over column headers
* See tooltip text in position
* Add a new column, hover, see text
* Add a column that has no text, hover, see no text
* Do same as above on update page

* Try learn more link
